mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -334,3 +345,22 @@ properties:
         name: 'expirationTime'
         description: |
           The expiration time of the schedule. The timestamp is an RFC3339 string.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'diskConsistencyGroupPolicy'
+    min_version: 'beta'
+    conflicts:
+      - 'snapshot_schedule_policy'
+      - 'group_placement_policy'
+      - 'instance_schedule_policy'
+    description: |
+      Replication consistency group for asynchronous disk replication.
+    send_empty_value: true
+    properties: 
+      - !ruby/object:Api::Type::Boolean
+        name: 'enabled'
+        default_value: true
+        immutable: true
+        description: |
+          Enable disk consistency on the resource policy. Defaults to `true`.
+    custom_expand: 'templates/terraform/custom_expand/disk_consistency_group_policy.erb'
+    custom_flatten: 'templates/terraform/custom_flatten/disk_consistency_group_policy.erb'
```

Comment:
This shouldn't need a custom flattener and expander... This seems 1:1 with the api representation no? I think we can omit this.

Reply:
Also removed the encoder now.